### PR TITLE
Honeywell Lyric Integration

### PR DESCRIPTION
**Pull request by timmo001**
## Proposed change

Docs for the new integration

## Type of change

- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands)

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/39695
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1855

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by timmo001** [comment]
Oops. Thanks bot!

File: source/_integrations/lyric.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   1. App Name: You can use any name here.
+   1. Callback URL: This needs to be the base URL of your Home Assistant instance + `/auth/external/callback`. For example: `http://192.168.1.123:8123/auth/external/callback`.
+
+Once your app is created, copy the `Consumer Key` and `Consumer Secret`. These will be your `client_id` and `client_secret` to add to your configuration:
+
+```yaml
+# Example configuration.yaml entry
+lyric:
+  client_id: myconsumerkey
+  client_secret: myconsumersecret
```
Comment (frenck) [inline source/_integrations/lyric.markdown]:
```suggestion
  client_id: MY_CONSUMER_KEY
  client_secret: MY_CONSUMER_SECRET
```

File: source/_integrations/lyric.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+You can then add the integration in the frontend.
+
+## Configuration
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **Honeywell Lyric**.
+Login with your Honeywell Lyric account and agree to the terms. The Honeywell Lyric integration will then be available.
```
Comment (frenck) [inline source/_integrations/lyric.markdown]:
```suggestion
Log in with your Honeywell Lyric account and agree to the terms. The Honeywell Lyric integration will then be available.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @timmo001!

Some small comments, could you take a look?

**Comment by frenck** [comment]
Oh, additionally, this might need a PR in our brand repository? As we should not rely on custom integration assets. So they need to be moved.

**Comment by timmo001** [comment]
> Oh, additionally, this might need a PR in our brand repository? As we should not rely on custom integration assets. So they need to be moved.

Looks like it already exists under core https://github.com/timmo001/brands/tree/master/core_integrations/honeywell

**Comment by frenck** [comment]
`honeywell` is a different integration, the domain key for this integration is `lyric` right?

image


Doesn't exist...

**Comment by timmo001** [comment]
> `honeywell` is a different integration, the domain key for this integration is `lyric` right?
> 
> image
> 
> 
> Doesn't exist...

Hmm strange how it shows up in the integrations in the frontend then.. 

Is it because it still has the word Honeywell in the title it still recognises it somehow?

**Comment by jasondefuria** [comment]
Look at the Lyric in Custom_Integrations and those files can be moved over. Here was my PR https://github.com/home-assistant/brands/pull/1771

**Review by bdraco — DISMISSED** [review]
LGTM 

https://deploy-preview-14409--home-assistant-docs.netlify.app/integrations/lyric/

Please address any additional comments that may come in after merge in a new PR.

**Comment by bdraco** [comment]
Actually Frenck's review is still fresh so this will need to wait for him to take a look over.

**Comment by frenck** [comment]
~~Fresh! Like today's salad! But I'll take a look tomorrow when cleaning things up in the morning~~

This is blocking something else, so let's get to it!

File: source/_integrations/lyric.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+The Honeywell Lyric component integrates the Lyric thermostat platform into Home Assistant.
+
+## Setup
+
+To set up this component, you will need to set up a developer account with Honeywell:
+
+1. Go to the [developer site](https://developer.honeywellhome.com) and register with an account.
+1. Next, create a [new app](https://developer.honeywellhome.com/user/me/apps/add) via the `My Apps` section.
+   1. App Name: You can use any name here.
+   1. Callback URL: This needs to be the base URL of your Home Assistant instance + `/auth/external/callback`. For example: `http://192.168.1.123:8123/auth/external/callback`.
```
Comment (frenck) [inline source/_integrations/lyric.markdown]: A small tweak, as the base URL doesn't matter anymore for OAuth 👍 

```suggestion
   1. Callback URL: This needs to be the URL of your Home Assistant instance + `/auth/external/callback`. For example: `http://192.168.1.123:8123/auth/external/callback`.
```

**Review by frenck — APPROVED** [review]

**Pull request merged**